_posts/2020-09-20-pulp-3.7.0-is-generally-available.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Pulp 3.7.0 is Generally Available!
+author: Melanie Corr
+tags:
+  - release
+---
+
+The Pulp team are pleased to announce the release of Pulp 3.7.0!
+
+
+## Export enhancements
+
+There have been a number of enhancements to the export process:
+
+* A progress report has been added to export tasks. [#6541](https://pulp.plan.io/issues/6541)
+
+* Improved handling of failed exports. Previously, when an export failed, the export's tarball was left on the filesystem. This has now been cleaned up so that nothing remains and there is a clean slate. [#7246](https://pulp.plan.io/issues/7246)
+
+* A `TOC_info` file has been added to point to the location of the TOC file during the export process. [#7221](https://pulp.plan.io/issues/7221)
+
+## Managing checksums
+
+`ALLOWED_CONTENT_CHECKSUMS` have been introduced so that users can specify and limit unwanted checksums in their environment. Note that plugins might not yet support `ALLOWED_CONTENT_CHECKSUMS` and that users should check plugin's release notes/docs for support before setting CONTENT_CHECKSUMS. [#5216](https://pulp.plan.io/issues/5216)
+
+## Enabling automatic file deletion for Filefield
+
+`FileField` has been enhanced with Django cleanup handlers so that files can be deleted from the `FileField`. With this release, when the delete() method is  evoked on the corresponding `FileField` model, automatic cleanup occurs. [#7316](https://pulp.plan.io/issues/7316)
```

Comment:
```
Pulp has added [django-cleanup](https://pypi.org/project/django-cleanup/) to automatically cleanup files for models with `FileField`. This also extends to models in plugins. Plugin writers can decorate their models with `@cleanup.ignore` to disable this.

Reply:
Decorate their models sounds like a lot more fun than it probably is. Thank you.